site/app/controllers/UserProfileController.php

```diff
@@ -145,6 +145,11 @@ public function changeProfilePhoto() {
         else {
             $meta = explode('.', $_FILES['user_image']['name']);
             $extension = $meta[1];
```

Comment:
While you're editing this code I think it would be a good change to edit how the file type is being read and instead use some of the helper functions in Submitty's `site/app/libraries/FileUtils.php` , mainly the `isValidImage` and `validateUploadedFiles` functions

Reply:
Thanks for pointing this out, didn't know that these functions existed.